src/ring_theory/roots_of_unity.lean

```diff
@@ -123,6 +116,16 @@ begin
   rw [subgroup.coe_pow, units.coe_pow],
 end
 
+lemma roots_of_unity.norm_one [normed_division_ring R] (φ : M →* R) {ζ : Mˣ} (h : ζ ^ (k : ℕ) = 1) :
```

Comment:
I think that instead of modifying the imports you can put this lemma in `analysis.normed.field.basic` or whatever. If you prefer to keep here (I don't have any strong preference) you should spell `h` as `ζ ∈ roots_of_unity k M`.

Reply:
I think the lemma belongs to this file so I changed the hypothesis as you suggested.